content/blog/accessing-hpe-data-fabric-s3-storage-from-spring-boot-s3-micro-service-deployed-in-k3s-cluster.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Accessing HPE Data Fabric S3 Storage from Spring Boot S3 Micro Service
+  deployed in K3S cluster
+date: 2021-09-13T08:13:40.734Z
+featuredBlog: true
+priority: 0
+author: Kiran Kumar Mavatoor
+authorimage: /img/hpe-dev-grommet-gremlin-rockin.svg
+tags:
+  - K3S
+  - Spring Boot
+  - HPE Data Fabric S3 Storage
+---
+S3 is a storage service offered by HPE Data Fabric Platform that offers scalable, secure, and well performing object storage. In this article, we will discuss as how to access HPE Data Fabric Object Store (S3) using Spring Boot S3 Micro Service application deployed in [K3S cluster](https://k3s.io/) and perform basic S3 operations like upload, list, delete etc. Below diagram gives the architecture overview 
+
+![Timeline
+
+Description automatically generated](blob:https://hpe-dev-portal.netlify.app/48945ed2-01ab-482a-b7f7-fd955b0b51bd)
+
+Figure 1:  Architecture overview of Spring Boot S3 Micro Service on K3S with HPE Data Fabric S3 Storage as back end.
```

Comment:
There is a problem with Figure 1

Reply:
Please use our guide here to learn best practices on how to add an image in the body of your blog post.
https://github.com/hpe-dev-incubator/hpe-dev-portal/blob/master/docs/ContributorGuide-v2.md#tips-and-trick-using-the-cms-editor

Reply:
uploaded the image